src/Symfony/Component/Finder/Comparator/Comparator.php

```diff
@@ -88,4 +105,16 @@ public function test($test)
 
         return $test == $this->target;
     }
+
+    /**
+     * @internal To be merged into the constructor in Symfony 6.0
```

Comment:
internal seems useless as private methods are always internal

Reply:
Makes sense. That annotation was more or less a reminder to inline the method in 6.0. I can remove it.